balancer/roundrobin/roundrobin_test.go

```diff
@@ -0,0 +1,65 @@
+package roundrobin
```

Comment:
Thanks for checking this in, but I mainly wanted to see the benchmarking code. I don't really desire to merge this into our main respository.

Reply:
The benchmark code is below in this file, the function name is BenchmarkPickConn100. or you want other benchmark test?

Reply:
so since you already see the benchmark code, now i just remove it now?

Reply:
Yes please.

Reply:
Can you delete this whole file please?

Reply:
> Can you delete this whole file please?

done